spec/spec_helper.rb

```diff
@@ -0,0 +1,99 @@
+require "simplecov"
+require "factory_girl_rails"
```

Comment:
I'm not 100% sure, but I think this line may need to be `require "factory_girl"`.